tests/wrapper/gmgetdate_test.php

```diff
@@ -36,8 +36,23 @@ public function test_phpbb_gmgetdate($timezone_identifier)
 		}
 
 		$expected = time();
+		$extra_offset = 0;
 
-		$date_array = phpbb_gmgetdate($expected);
+		// Handle DST switches in progress
+		if ($timezone_identifier)
+		{
+			$yesterday = $expected - 86400;
+			$tomorrow = $expected + 86400;
+			$identifier_tz = new DateTimeZone($timezone_identifier);
+
+			$yesterday_date = new DateTime("@{$yesterday}");
+			$yesterday_date = $yesterday_date->setTimezone($identifier_tz);
+			$tomorrow_date = new DateTime("@{$tomorrow}");
```

Comment:
I think you can directly put `new DateTime("tomorrow");`, same for yesterday

Reply:
Decided to remove this as it's not fully working